src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -442,6 +442,10 @@ protected function get_available_actions() {
 			$rels[] = 'https://api.w.org/action-publish';
 		}
 
+		if ( current_user_can( 'edit_css' ) ) {
+			$rels[] = 'https://api.w.org/action-edit-css';
+		}
```

Comment:
There's an [unanswered question from the Gutenberg PR](https://github.com/WordPress/gutenberg/pull/46815#discussion_r1058902276)

>I couldn't find documentation on how these URLs are generated/created. So I need to double-check if `https://api.w.org/action-edit-css` is the correct format.

Pinging @TimothyBJacobs @spacedmonkey for review.

Reply:
All similar action URLs lead to a 404 page - `https://api.wordpress.org/action-unfiltered-html`.

Reply:
@TimothyBJacobs, @spacedmonkey, can you confirm that it's okay to use this URL?

Reply:
Sorry for missing this the first time round. The URLs don't actually point anywhere. That URL is fine to use.

Reply:
Thank you, @TimothyBJacobs!